modules/cluster-logging-collector-log-forward-loki.adoc

```diff
@@ -29,15 +33,17 @@ spec:
      url: http://loki.insecure.com:3100 <6>
      loki:
         tenantKey: kubernetes.namespace_name
-        labelKeys: kubernetes.labels.foo
+        labelKeys:
```

Comment:
The indentation seems to be different for different parts of this example file. I think it would make sense to use the same indent everywhere (can't create a suggestion for all lines, unfortunately):

```yaml
apiVersion: logging.openshift.io/v1
kind: ClusterLogForwarder
metadata:
  name: <log_forwarder_name> <1>
  namespace: <log_forwarder_namespace> <2>
spec:
  serviceAccountName: <service_account_name> <3>
  outputs:
  - name: loki-insecure <4>
    type: "loki" <5>
    url: http://loki.insecure.com:3100 <6>
    loki:
      tenantKey: kubernetes.namespace_name
      labelKeys:
      - kubernetes.labels.foo
  - name: loki-secure <7>
    type: "loki"
    url: https://loki.secure.com:3100
    secret:
      name: loki-secret <8>
    loki:
      tenantKey: kubernetes.namespace_name <9>
      labelKeys:
      - kubernetes.labels.foo <10>
  pipelines:
  - name: application-logs <11>
    inputRefs:  <12>
    - application
    - audit
    outputRefs: <13>
    - loki-secure
```

Reply:
Thanks so much